### PR TITLE
Bug fixes for gaussianfg and chime freq option

### PR DESCRIPTION
**Pull request by cahofer**
frequencies need endpoint false when using in combination with channel-bin when you want to match the map frequencies with the frequencies in the CHIME telescope class.
added another option - now we have 'centre' 'chime' and 'edge'

### CODE REVIEW

File: cora/scripts/makesky.py [hunk 1]
```diff
@@ -12,9 +12,9 @@
 @click.option('--channels', help='Select a range of frequency channels',
               type=(int, int), metavar='CSTART CSTOP', default=(None, None))
 @click.option('--channel-bin', help='If set, average over BIN channels', metavar='BIN', type=int, default=1)
-@click.option('--freq-mode', type=click.Choice(['centre', 'edge']), default='centre',
-              help=('Choose whether FSTART and FSTOP are the very edges of the band, '+
-                    'or whether they are the centre frequencies (default: centre).'))
+@click.option('--freq-mode', type=click.Choice(['centre', 'chime', 'edge']), multiple=True, default='centre',
+              help=('Choose whether FSTART and FSTOP are the centre frequencies, the frequencies for ' +
+                    'the CHIME telescope or the very edges of the band, (default: centre).'))
```
Comment (jrs65) [inline cora/scripts/makesky.py]: It's not clear why we need this option. Can't you get the same thing by just setting `--channels 0 1024`?

File: cora/scripts/makesky.py [hunk 2]
```diff
@@ -180,7 +183,9 @@ def gaussianfg(ctx):
 
     # Set frequency parameters
     fsyn.frequencies = ctx.obj.freq
+    nfreq = len(fsyn.frequencies)
```
Comment (jrs65) [inline cora/scripts/makesky.py]: Thanks. These bug fixes look good.

File: cora/scripts/makesky.py (hunk 1 above)
Comment (cahofer) [reply cora/scripts/makesky.py]: True. I missed that and reverted that change.

**Comment by jrs65** [comment]
Awesome. Thanks!!

**Pull request merged**